project/plugins.sbt

```diff
@@ -4,7 +4,7 @@ addSbtPlugin("com.jsuereth"      % "sbt-pgp"         % "2.0.1")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-osgi"        % "0.9.5")
 addSbtPlugin("io.spray"          % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("org.scalameta"     % "sbt-mdoc"        % "2.2.3")
-addSbtPlugin("org.scalariform"   % "sbt-scalariform" % "1.8.3")
+addSbtPlugin("org.scalameta"     % "sbt-scalafmt"    % "2.4.0")
```

Comment:
Can you also make this change, and respective code format, in the example project?